### PR TITLE
Flesh out Symptom Screen

### PR DESCRIPTION
**Pull request by AshlandWest**

### CODE REVIEW

**Comment by AshlandWest** [comment]
All rebase are belong to us

File: App.js [hunk 1]
```diff
@@ -5,10 +5,17 @@ import React, { useState } from "react";
 import { Platform, StatusBar, StyleSheet, View, I18nManager as RNI18nManager } from "react-native";
 import { Ionicons } from "@expo/vector-icons";
 import { Provider } from "react-redux";
+<<<<<<< HEAD
```
Comment (BranescuMihai) [inline App.js:8]: Some conflict merging indicators were forgotten here, please remove them

**Review by BranescuMihai — CHANGES_REQUESTED** [review]
From the number of files changes, I feel that the first commit is somehow actually a rebase from your feature `onto` master instead of `from`. This will rewrite all project history and is generally[ not a good idea](https://www.atlassian.com/git/tutorials/merging-vs-rebasing#the-golden-rule-of-rebasing).

Since I'm assuming this feature is actually a new screen, I strongly recommend to create a new branch from master, and just add the new changes there, as fixing this would require you look into the git logs and reverting some steps, which is quite advanced

**Comment by luxx** [comment]
Closing this out, feel free to reopen a new one once the conflicts are fixed

**Pull request closed without merging**